### DISCUSSION TITLE
trouble with awus036h

### DISCUSSION THREAD

**Discussion by woejefe** [svpcom/wfb-ng#298] [Questions] [answered]
I know this isnt on approved hardware list but i dont see why it cant work.... It shows iw putting the interface in monitor mode but getting error setting channel

` sudo systemctl status wifibroadcast@gs
● wifibroadcast@gs.service - WFB profile gs
     Loaded: loaded (/lib/systemd/system/wifibroadcast@.service; enabled; vendor preset: enabled)
     Active: active (running) since Sun 2023-05-28 18:14:27 BST; 4s ago
   Main PID: 714 (wfb-server)
      Tasks: 1 (limit: 1934)
        CPU: 2.205s
     CGroup: /system.slice/system-wifibroadcast.slice/wifibroadcast@gs.service
             └─714 /usr/bin/python3 /usr/bin/wfb-server gs wlan0

May 28 18:14:29 GSpi wfb-server[714]: /etc/wifibroadcast.cfg
May 28 18:14:29 GSpi wfb-server[714]: [proxy._check_rc #info] # iw reg set BO
May 28 18:14:29 GSpi wfb-server[714]: [proxy._check_rc #info] # ip link set wlan0 down
May 28 18:14:29 GSpi wfb-server[714]: [proxy._check_rc #info] # iw dev wlan0 set monitor otherbss
May 28 18:14:31 GSpi wfb-server[714]: [proxy._check_rc #info] # ip link set wlan0 up
May 28 18:14:31 GSpi wfb-server[714]: [server.init_wlans #error] kernel reports: (extension) channel is disabled
May 28 18:14:31 GSpi wfb-server[714]: command failed: Invalid argument (-22)
May 28 18:14:31 GSpi wfb-server[714]: [common.abort_on_crash #info] Stopping reactor due to fatal error: RC 234: iw dev wlan0 set channel 11>
May 28 18:14:31 GSpi wfb-server[714]: [base.EPollReactor.mainLoop #info] Main loop terminated.
May 28 18:14:31 GSpi wfb-server[714]: [server.main #info] Exiting with code 1
`

kernel reports channel is disabled... not sure how to fix

**Comment by svpcom** [comment]
What driver does it use?

**Reply by woejefe** [reply]
i was using the default installed driver

**Reply by woejefe** [reply]
i was looking into installing rtl8187 driver. just not sure what to use

**Reply by svpcom** [reply]
`ethtool -i wlan0`

**Reply by woejefe** [reply, marked as answer]
i couldnt get awus036h with rtl8187 driver.... ended up using little edimax 7811-un on GS.

All good now

**Reply by woejefe** [reply]
My setup is:
pi 3 b+  edimax 7811-un on GS
pi 3 b+ with picam and awus052nh on RC car

I ran apt-get update & upgrade and it installed the correct drivers for my dongles... may still try to get awus036h to work because the range is great

**Reply by svpcom** [reply]
rtl8187 is not supported by https://github.com/svpcom/rtl8812au
Most of standard wifi drivers doesn't support injection
